airgun/widgets.py

```diff
@@ -2188,3 +2190,44 @@ def fill(self, item):
     def read(self):
         """read column updated value"""
         return self.column_value.read()
+
+
+class AuthSourceAggregateCard(AggregateStatusCard):
+    """This is a customizable card widget which has the title, count and kebab widget
+     Example html representation::
+         <div class="card-pf-body">
```

Comment:
I think you need empty line between line ending with `::` and code.
That should fix travis error, which is a little cryptic.